### DISCUSSION TITLE
Diagrama de requerimientos

### DISCUSSION THREAD

**Discussion by LuceroLuciano** [Launch-X-Latam/MisionFrontEnd#136] [Show and tell]
# Hola a todxs explorers

Aquí un diagrama que hice para el proyecto de abogabot
[Diagrama Abogabot](https://drive.google.com/file/d/1rp1WDu7ZyNx6KkCs0_5FdsvBbY4QvZxG/view?usp=sharing)
Agradecería mucho su feedback. ✨

**Comment by josmanyel** [comment]
Todavía no lo reviso bro porque es de noche y estoy revisando todos los discussions, pero lo haré y te daré mi feedback ok voy paso a paso

**Comment by josmanyel** [comment]
Gracias por compartir

**Comment by doguedogue** [comment]
Se ve genial, está hecho con https://app.diagrams.net/ (antes [draw.io](https://www.draw.io/)) 🚀🚀🚀
